### PR TITLE
feat: Add missing `DROP VIEW` statement, ignore additional unsupported statements in Emulator

### PR DESCRIPTION
**Pull request by abdel**
- Adds missing `DROP VIEW` to the supported DDL statements
- Ignore `CREATE VIEW`, `CREATE OR REPLACE VIEW`, `CREATE ROLE`, `DROP ROLE`, `GRANT` and `REVOKE` DDL statements in Emulator as they are not supported.

### CODE REVIEW

File: src/loader/parse-migrations.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
 
 function getStatementType(stm: Statement): StatementType {
   if (
-    /^(CREATE TABLE|ALTER TABLE|DROP TABLE|CREATE INDEX|ALTER INDEX|DROP INDEX|CREATE VIEW|CREATE OR REPLACE VIEW|CREATE ROLE|DROP ROLE|GRANT|REVOKE)/i.test(
+    /^(CREATE TABLE|ALTER TABLE|DROP TABLE|CREATE INDEX|ALTER INDEX|DROP INDEX|CREATE VIEW|CREATE OR REPLACE VIEW|DROP VIEW|CREATE ROLE|DROP ROLE|GRANT|REVOKE)/i.test(
```
Comment (abdel) [inline src/loader/parse-migrations.ts:13]: I forgot to add `DROP VIEW` as a supported statement

File: src/loader/parse-migrations.ts [hunk 2]
```diff
@@ -31,7 +31,10 @@ function validateFileName(name: string): void {
 
 function isStatementSupportedByEmulator(statement: string): boolean {
   const notSupportedPatterns: ((statement: string) => boolean)[] = [
-    (statement) => /^alter table [a-z][\d\w_]{0,128} add row deletion policy .+$/i.test(statement),
+    (statement) =>
+      /^(alter table [a-z][\d\w_]{0,128} add row deletion policy .+$|create view|create or replace view|drop view|create role|drop role|grant|revoke)/i.test(
```
Comment (abdel) [inline src/loader/parse-migrations.ts:35]: Tested locally with the emulator to check all the VIew DDL statements, as well as the newly added role-related ones which were also unsupported in Emulator and errored.

**Comment by mdovhopo** [comment]
Looks very good. Merging this.

**Comment by mdovhopo** [comment]
had a busy weekend/start of the week :)

**Pull request merged**